etc/settings.yaml.in

```diff
@@ -50,29 +51,29 @@ parameters:
   # the visualizations or the linear regression calculations.
   reporting:
     mutation-coverage-threshold: 90
-    mutation-depth-threshold: 100
-    
 
-# Control the deconvolution method used.
-# Possible options are:
-#   weighted_rlm(default): 
-#     Robust linear regression with bulk mutation abundances weighted per
-#     variant by the inverse of the proportion of detected mutations to known
-#     mutations. This biases the abundance of variants with high proportions
-#     towards higher values, and coversely biases the abundence of variants with
-#     low proportions towards lower values. Robust linear regression is
-#     implemented in the `MASS` package.
-#   rlm:
-#     Robust linear regression as implemented in the `MASS` package, executed
-#     via the `deconvR` package.
-#   nnls:
-#     Non-negative least squares (`deconvR`).
-#   qp:
-#     Quadratic programming (`deconvR`).
-#   svr:
-#     Support vector regression (`deconvR`).
-deconvolution:
-  method: weighted_rlm
+
+  deconvolution:
+    # Control the deconvolution method used.
+    # Possible options are:
+    #   weighted_rlm(default):
+    #     Robust linear regression with bulk mutation abundances weighted per
+    #     variant by the inverse of the proportion of detected mutations to
+    #     known mutations. This biases the abundance of variants with high
+    #     proportions towards higher values, and coversely biases the abundence
+    #     regression is of variants with low proportions towards lower values.
+    #     Robust linear implemented in the `MASS` package.
+    #   rlm:
+    #     Robust linear regression as implemented in the `MASS` package, executed
+    #     via the `deconvR` package.
+    #   nnls:
+    #     Non-negative least squares (`deconvR`).
+    #   qp:
+    #     Quadratic programming (`deconvR`).
+    #   svr:
+    #     Support vector regression (`deconvR`).
```

Comment:
I'm not a fan of moving the comments inside the block, but I don't feel strongly enough about this to argue :)

Reply:
Hmm I removed all the comments, as everything should be explained in the docs anyways...